packages/elements/src/pagination/index.ts

```diff
@@ -574,6 +579,86 @@ export class Pagination extends BasicElement {
     this.notifyValueChange();
   }
 
+  protected hasNextPage (page: number): boolean {
```

Comment:
Function comments ? and I think it is unnecessary to extract this as a new function.

Reply:
It seems a simple function but it is useful for refactoring and remove a duplicate code the pagination in the next sprint